specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/inference.json

```
@@ -0,0 +1,1443 @@
+        "x-ms-examples": {
+          "Get Endpoints": {
+            "$ref": "./examples/getInferenceEndpoints.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/PaginationParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The response includes a paginated array of Machine Learning Inference Endpoints.",
+            "schema": {
+              "$ref": "#/definitions/PaginatedInferenceEndpointsList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the request failed.",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/inferenceEndpoints/{inferenceEndpointName}": {
+      "get": {
+        "tags": [
+          "MachineLearningInferenceEndpoints"
+        ],
+        "operationId": "MachineLearningInferenceEndpoint_Get",
+        "description": "Gets Details in specified Inference Endpoint.",
+        "x-ms-examples": {
+          "Get Endpoint": {
+            "$ref": "./examples/getInferenceEndpoint.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/PaginationParameter"
+          },
+          {
+            "$ref": "#/parameters/InferenceEndpointNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The response includes a Machine Learning Inference Endpoint based on InferenceEndpointName, if any.",
+            "schema": {
+              "$ref": "#/definitions/InferenceEndpointResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the request failed.",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "MachineLearningInferenceEndpoints"
+        ],
+        "operationId": "MachineLearningInferenceEndpoint_Delete",
```

Align with the List operationId
```suggestion
        "operationId": "MachineLearningInferenceEndpoints_Delete",
```